config/initializers/devise.rb

```
@@ -0,0 +1,313 @@
+# breaking changes in upgrades (i.e., in the event that future versions of
+# Devise change the default values for those options).
+#
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '2f563b13f38926c474d574175c02d3368a69f9c70ca971bd79e4ddc42e846fbf28c1d343c2b8d7791e8bef5ce054034475e022fc7ddf01db17effa1a718be622'
```

RuboCop (1.75.5)

[convention] Line is too long. [154/120]

(Layout/LineLength)